src/shared-components/tag/style.js

```diff
@@ -0,0 +1,74 @@
+import styled, { css } from 'react-emotion';
+
+import { style as TYPOGRAPHY_STYLE } from '../typography';
+import { COLORS, SPACING } from '../../constants';
+
+const closedStyle = css`
+  background-color: ${COLORS.statusGreyBackground};
+  border-color: ${COLORS.statusGreyBorder};
+  span {
```

Comment:
[note] i **think** if you rearrange, you can actually interpolate `TagText` here to make this more specific

Reply:
Yup that makes sense. Will update 👍

Reply:
@snags88 @smmullen90 I believe we have `badge` already. 

It's this: 
![image](https://user-images.githubusercontent.com/28582716/49393803-1d0bf000-f6e7-11e8-9263-f2eed06acedd.png)

It is similar though, wonder if we could have it as two separate types of `badges`. 

I'll throw it up in the slack channel for people to discuss!